requirements.in

```diff
@@ -25,6 +25,7 @@ gunicorn==20.1.0
 html5lib==1.1
 mailchimp3==3.0.15
 mutagen==1.45.1
+packaging==24.0
```

Comment:
This should fix https://github.com/oppia/oppia/issues/20747 -- setuptools (>71) incompatibility with packaging (<24)